### ISSUE TITLE
Auto épület javítás

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#16]
```
Game version: 0.92


Operating System: (e.g., Windows 7 x86, Windows XP 64-bit)

win7 64-bit
Java runtime version: (run java -version)
7

Installed using the Launcher? (yes, no)

yes
What steps will reproduce the problem?
1.Épületek automatikus javítás bekapcsolása
2.Akkor is fogy a credit, ha nincs sérült épület






```

Original issue reported on code.google.com by `Jozsef.T...@gmail.com` on 14 Aug 2011 at 8:14

**Comment by GoogleCodeExporter** [comment]
```
Kösz. A v0.93 majd tartalmazza a javítást.

The issue in English:

When Auto-Repair is turned on in the settings screen, it drains money even when 
there are no damaged buildings. Fixed in v0.93.
```

Original comment by `akarn...@gmail.com` on 14 Aug 2011 at 10:08
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.93.100
```

Original comment by `akarn...@gmail.com` on 17 Aug 2011 at 10:56
- Changed state: **Fixed**
